proxy/proxy.go

```diff
@@ -1184,7 +1184,9 @@ func (p *Proxy) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 				Duration:     time.Since(ctx.startServe),
 			}
 
-			logging.LogAccess(entry)
+			additionalData, _ := ctx.stateBag[al.AccessLogAdditionalDataKey].(map[string]interface{})
```

Comment:
Do we need to cast? As far as I remember, it’s already the type map[string]interface{}.
On my phone so can not look easily 😇

Reply:
No, the "value" of the StateBag is `interface{}`, so if not casted, compilation fails.

Reply:
yes, true you are right!